src/snowflake/cli/api/metrics.py

```diff
@@ -84,24 +85,52 @@ class for holding metrics span data and encapsulating related operations
     START_TIME_KEY: ClassVar[str] = "start_time"
     EXECUTION_TIME_KEY: ClassVar[str] = "execution_time"
     ERROR_KEY: ClassVar[str] = "error"
+    COUNT_SPANS_IN_SUBTREE_KEY: ClassVar[str] = "count_spans_in_subtree"
+    SPAN_DEPTH_KEY: ClassVar[str] = "span_depth"
+    # denotes whether direct children were pruned from telemetry payload
+    TRIMMED_KEY: ClassVar[str] = "trimmed"
```

Comment:
nit: pruned or trimmed? You're using both for the same thing.

Reply:
I thought they were synonyms, checked Google and seems trimmed works better so I kept that one 😓